RecoMET/METFilters/plugins/CSCSuperTightHalo2016Filter.cc

```diff
@@ -0,0 +1,43 @@
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/global/EDFilter.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "DataFormats/METReco/interface/BeamHaloSummary.h"
+
+class CSCSuperTightHalo2016Filter : public edm::global::EDFilter<> {
```

Comment:
CSCSuperTightHalo2016Filter and CSCTightHalo2016Filter can be merged into more flexible filter module.